.gitignore

```diff
@@ -80,3 +80,35 @@ ircmaxell/random-lib/composer.lock
 ircmaxell/random-lib/phpunit.xml.dist
 
 icewind/streams/tests
+
+guzzlehttp/guzzle/build/
+guzzlehttp/guzzle/docs/
+guzzlehttp/guzzle/tests/
+guzzlehttp/guzzle/.gitignore
+guzzlehttp/guzzle/.travis.yml
+guzzlehttp/guzzle/CHANGELOG.md
+guzzlehttp/guzzle/composer.json
+guzzlehttp/guzzle/LICENSE
+guzzlehttp/guzzle/MAKEFILE
+guzzlehttp/guzzle/phpunit.xml.dist
+guzzlehttp/guzzle/README.md
+guzzlehttp/guzzle/UPGRADING.md
+guzzlehttp/ringphp/docs/
+guzzlehttp/ringphp/tests/
+guzzlehttp/ringphp/.gitignore
+guzzlehttp/ringphp/.travis.yml
+guzzlehttp/ringphp/CHANGELOG.md
+guzzlehttp/ringphp/LICENSE
+guzzlehttp/ringphp/Makefile
+guzzlehttp/ringphp/README.rst
+guzzlehttp/ringphp/composer.json
+guzzlehttp/ringphp/phpunit.xml.dist
+guzzlehttp/streams/tests/
+guzzlehttp/streams/.gitignore
+guzzlehttp/streams/.travis.yml
+guzzlehttp/streams/CHANGELOG.rst
+guzzlehttp/streams/LICENSE
+guzzlehttp/streams/Makefile
+guzzlehttp/streams/README.rst
+guzzlehttp/streams/composer.json
+guzzlehttp/streams/phpunit.xml.dist
```

Comment:
Can you create a dedicated issue? – This feels out-of-scope for me.
(as in: "I really don't know what third-party apps we ship that abuse the name `docs` for something else.")